app/config/parameters.yml.dist

```diff
@@ -38,6 +38,7 @@ parameters:
     dismissible_info_bars: {}
     cache_control: public, max-age=300, stale-while-revalidate=300, stale-if-error=86400
     feature_xpub: true
+    feature_rds: true
```

Comment:
Should add `rds_articles` to this too. We should use a futureproof syntax (to allow for the separate download URI), so something like:

```yaml
rds_articles:
    '30274':
        display: https://repro.elifesciences.org/example.html
```